src/views/Dashboard/DashboardTable.tsx

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+import { ethers } from "ethers";
+import { CellProps, Column, Renderer } from "react-table";
+import { useAllReserveTokensWithData } from "../../queries/lendingReserveData";
+import { useAssetPriceInDai } from "../../queries/assetPriceInDai";
+import { useDepositAPY } from "../../queries/depositAPY";
+import {
+  BasicTableRenderer,
+  SortedHtmlTable,
+  TableRenderer,
+} from "../../utils/htmlTable";
+import { Box, Text } from "@chakra-ui/layout";
+import { Button, Flex, Switch } from "@chakra-ui/react";
+import { TokenIcon } from "../../utils/icons";
+import { useUserAssetBalance } from "../../queries/userAssets";
+import ColoredText from "../../components/ColoredText";
+
+export enum DashboardTableType {
+  Deposit = "DEP",
+  Borrow = "BORR",
```

Comment:
There's no reason to abbreviate these; string operations won't change anything significantly when compared to JS overhead.
```suggestion
  Deposit = "Deposit",
  Borrow = "Borrow",
```